content/en/coec/index.adoc

```diff
@@ -0,0 +1,53 @@
+---
+title: "Code of Ethics and Conduct"
+description: "ACL Special Interest Group on Turkic Languages Code of Ethics and Conduct"
+draft: false
+toc: true
+images: []
+contributors: []
+---
+
+:toc:
+
+== Introduction
+
+SIGTURK is an ACL Special Interest Group on Turkic Languages that aims to foster community brainstorming and collaboration on events and resources for the public good in open source. As these processes involve sync and async communication, this document includes professional standards all participants of SIGTURK and SIGTURK-related dialogue must uphold to promote formal reliability of SIGTURK outputs, respect, competence, responsibility, and integrity while recognizing the rapidly changing society and the necessity to exercise more considerable caution in this spirit by every participant.
+
+SIGTURK also recognizes the following documents as authoritative, and their reading should interpret the authoring organization's name as SIGTURK under all respect to the license of the individual page:
+
+* ACL Code of Ethics: https://www.aclweb.org/portal/content/acl-code-ethics
+* ACL Anti-Harassment Policy: https://www.aclweb.org/adminwiki/index.php?title=Anti-Harassment_Policy
+* ACM Code of Ethics and Professional Conduct: https://www.acm.org/code-of-ethics
+* W3C Code of Ethics and Professional Conduct: https://www.w3.org/Consortium/cepc
+* GitHub Community Code of Conduct: https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct
+* RISC-V Community Code of Conduct: https://riscv.org/community/community-code-of-conduct/
+* SIL Basic Principles of Ethical Research in Fieldwork: https://www.sil.org/about/discover/academic-research
+
+These organizations and codes of ethics and conduct are model documents for SIGTURK with authoritative relevance.
+
+== Pledge
```

Comment:
I somehow dislike usage of the word "pledge". I would prefer something like this as in https://www.acm.org/code-of-ethics:
```
4.1 Uphold, promote, and respect the principles of the Code.

The future of computing depends on both technical and ethical excellence. Computing professionals should adhere to the principles of the Code and contribute to improving them. Computing professionals who recognize breaches of the Code should take actions to resolve the ethical issues they recognize, including, when reasonable, expressing their concern to the person or persons thought to be violating the Code.
```

Reply:
Pledge is a common term for such a section, do we really need to avoid it? The closest term I can think of is "commitment" but I don't see it in use really. Title structure of ACM is a bit different, from there maybe "compliance" can be an option but I believe Pledge has overwhelming vote in sources of such text